### PR TITLE
Update list of validators and format to display

### PR DESCRIPTION
**Pull request by Rub21**
screenshot from 2016-11-09 18 29 44

@geohacker , could you take a look on the improvement to display the validators.

### CODE REVIEW

**Comment by Rub21** [comment]
the md file too: https://github.com/osmlab/osmlint/blob/doc/validators.md

File: createValidator.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+### How do I create a new validator?
```
Comment (geohacker) [inline createValidator.md]: @Rub21 - let's call this file CONTRIBUTING.md

**Comment by geohacker** [comment]
Hey @Rub21 - thank you for the PR.

* Let's call createValidators.md as CONTRIBUTING.md
* I'm not sure if we need that shell script to print the help - this plus the validators.md file becomes redundant and adds one more place to update when we are contributing validators. 
* Can we just print the validators.md file?

**Comment by Rub21** [comment]
@geohacker 
> Let's call createValidators.md as CONTRIBUTING.md

done!
>I'm not sure if we need that shell script to print the help - this plus the validators.md file becomes redundant and adds one more place to update when we are contributing validators.
> Can we just print the validators.md file?

Yes, could be redundant, lets print the  validators.md , Done!

**Pull request merged**